features/support/hooks.js

```diff
@@ -48,9 +48,10 @@ registerHandler('AfterFeatures', ([{options}]) => {
   reporter.generate(options)
 })
 
-registerHandler('ScenarioResult', function (scenario) {
+registerHandler('ScenarioResult', async function (scenario) {
   if (scenario.status === 'failed') {
-    driver.takeScreenshots1('Screenshots1')
+    const filename1 = await driver.getCurrentUrlName()
```

Comment:
Just wondering why you called this **filename1** instead of **filename**

Reply:
changed it to filename